src/elements/Price.php

```diff
@@ -393,6 +398,73 @@ protected static function defineDefaultTableAttributes(string $source): array
         ];
     }
 
+    /**
+     * @inheritdoc
+     */
+    protected static function defineCardAttributes(): array
+    {
+        return array_merge(parent::defineCardAttributes(), [
+            'primaryCurrency' => [
+                'label' => Craft::t('stripe', 'Primary Currency'),
```

Comment:
"Primary Currency" is missing from the translations file.

Reply:
Thanks! Done.